src/traces/histogram/attributes.js

```diff
@@ -71,6 +71,46 @@ module.exports = {
         ].join(' ')
     },
 
+    cumulative: {
```

Comment:
It would be nice to add one image mock. Maybe one that combines a `currentbin: 'include'` and `currentbin: 'exclude'`  traces like in:

![image](https://cloud.githubusercontent.com/assets/6675409/21992171/76ad4b02-dbe3-11e6-893e-db3295383ec2.png)